modules/tinymce/src/plugins/table/test/ts/module/test/TableModifiersTestUtils.ts

```diff
@@ -172,6 +172,36 @@ const pAssertStyleCanBeToggledOnAndOffWithoutCheckmarks = async (editor: Editor,
   await pAssertStyleCanBeToggledWithoutCheckmarks(editor, options, 'menuitem');
 };
 
+const makeCell = (type: string, content: string, hasScope?: boolean, isSelected?: boolean, isStart?: boolean, isEnd?: boolean) => {
```

Comment:
I just thought I'd note that I felt this made the tests more difficult to read than just inlining the selected attributes like we do elsewhere.

This is just so difficult to read:
`makeCell('td', '1-1', false, true, false, true)`

Reply:
I found it a lot easier to read and write that, rather than having the long pieces of attributes everywhere myself, not to mention having to put them everywhere.
But if people thinks it's easier to read all the selection-stuff, I'll put it there instead.

Reply:
I'm with James on this one, this is hard to read and either passing an object (so you can see the names) or just passing the values would be better.